WinUIGallery/ItemsPageBase.cs

```diff
@@ -76,14 +76,14 @@ protected void OnItemGridViewKeyDown(object sender, KeyRoutedEventArgs e)
                     SearchRoot = this.XamlRoot.Content
                 };
                 var nextElement = FocusManager.FindNextElement(FocusNavigationDirection.Up, options);
-                if (nextElement?.GetType() == typeof(Microsoft.UI.Xaml.Controls.NavigationViewItem))
-                {
-                    NavigationRootPage.GetForElement(this).PageHeader.Focus(FocusState.Programmatic);
-                }
-                else
-                {
-                    FocusManager.TryMoveFocus(FocusNavigationDirection.Up);
-                }
+                //if (nextElement?.GetType() == typeof(Microsoft.UI.Xaml.Controls.NavigationViewItem))
+                //{
+                //    NavigationRootPage.GetForElement(this).PageHeader.Focus(FocusState.Programmatic);
+                //}
+                //else
+                //{
+                //    FocusManager.TryMoveFocus(FocusNavigationDirection.Up);
+                //}
```

Comment:
Intentional?

Reply:
The PageHeader was not in use.. any idea what the intended behavior was here?

Reply:
Oh nvm.. was thinking about the landings page.

Reply:
Oh no.. wait.. this was only used on the landings page and controls overviews where the PageHeader was no longer in use. So this can be removed all together?

Reply:
Yes I think we can remove that.